packages/wasm-dpp/src/state_transition/state_transition_factory.rs

```diff
@@ -38,6 +38,47 @@ impl From<StateTransitionFactory<ExternalStateRepositoryLikeWrapper, BlsAdapter>
     }
 }
 
+impl StateTransitionFactoryWasm {
+    pub fn state_transition_wasm_from_factory_result(
+        result: Result<StateTransition, ProtocolError>,
+    ) -> Result<JsValue, JsValue> {
+        match result {
+            Ok(state_transition) => match state_transition {
+                StateTransition::DataContractCreate(st) => {
+                    Ok(DataContractCreateTransitionWasm::from(st).into())
+                }
+                StateTransition::DataContractUpdate(st) => {
+                    Ok(DataContractUpdateTransitionWasm::from(st).into())
+                }
+                StateTransition::IdentityCreate(st) => {
+                    Ok(IdentityCreateTransitionWasm::from(st).into())
+                }
+                StateTransition::IdentityUpdate(st) => {
+                    Ok(IdentityUpdateTransitionWasm::from(st).into())
+                }
+                StateTransition::IdentityTopUp(st) => {
+                    Ok(IdentityTopUpTransitionWasm::from(st).into())
+                }
+                StateTransition::DocumentsBatch(st) => {
+                    Ok(DocumentsBatchTransitionWASM::from(st).into())
```

Comment:
I know this isn't the subject for this PR, but can you please rename it to `Wasm`? 😅

Reply:
Done